src/scss/components/checkbox/_index.scss

```diff
@@ -120,11 +120,11 @@ hx-checkbox-control {
 
     &:checked,
     &:indeterminate {
-      color: $cyan-900;
+      color: var(--hxCheckbox-indeterminate, $cyan-900);
 
       ~ label > hx-checkbox {
-        border-color: $cyan-900;
-        color: $cyan-900;
+        border-color: var(--hxCheckbox-indeterminate-label-bordercolor, $cyan-900);
+        color: var(--hxCheckbox-indeterminate-label, $cyan-900);
```

Comment:
please change to var(--hxCheckbox-indeterminate-label-color, $cyan-900);

Reply:
Changes have been done, Please review